config/webpack/webpack.config.js

```diff
@@ -70,6 +71,13 @@ module.exports = function (dirname, config = {}) {
             new CircularDependencyPlugin({
                 exclude: /(node_modules|examples)\/./,
                 failOnError: false // https://github.com/nodejs/readable-stream/issues/280#issuecomment-297076462
+            }),
+            new GlobalizePlugin({
+                production: false,
+                developmentLocale: "en",
+                supportedLocales: ["en", "es", "fr"],
+                messages: "../../src/i18n/[locale].json",
```

Comment:
All changes were made. The only thing is to be able to split the folder containing all messages for each language into smaller files under each application folder. We can do this when we will know how to read and load the messages for each application. For now, I suggest we keep the i18n folder at the root with all changes .